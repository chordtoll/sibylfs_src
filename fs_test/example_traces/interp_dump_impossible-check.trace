# processing file 'example_traces/simple_dump-int.trace' ...
        @type trace
     1: dump-result /impossible
           "/"|D|0
           "/oooOOOOOoooo"|D|1
        end dump-result
