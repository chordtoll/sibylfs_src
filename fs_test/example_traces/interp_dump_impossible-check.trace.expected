# processing file 'example_traces/interp_dump_impossible-check.trace' ...
        # processing file 'example_traces/simple_dump-int.trace' ...
        @type trace
     3: dump-result "/impossible"
           "/"|D|0
           "/oooOOOOOoooo"|D|1
        end dump-result
# 
# Error:      3: dump-result "/impossible"
#            "/"|D|0
#            "/oooOOOOOoooo"|D|1
#         end dump-result
#    execution of dump failed

# trace not accepted
# processing file 'example_traces/interp_dump_impossible-check.trace' failed
